### PR TITLE
improve get description

### PR DESCRIPTION
**Pull request by ChiuWang**
ref: #60210

/assign @deads2k 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/resource/get.go [hunk 1]
```diff
@@ -87,9 +87,8 @@ var (
 		namespace unless you pass --all-namespaces.
 
 		This command will hide resources that have completed, such as pods that are
```
Comment (deads2k) [inline pkg/kubectl/cmd/resource/get.go]: The advice here has always been wrong since it was only pods.  Remove the first sentence entirely please.

Comment (charrywanganthony) [reply pkg/kubectl/cmd/resource/get.go]: @deads2k Modified.

**Pull request merged**